tests/suites/test_suite_aria.function

```diff
@@ -16,6 +16,186 @@
  * END_DEPENDENCIES
  */
 
+/* BEGIN_CASE depends_on:MBEDTLS_CHECK_PARAMS:!MBEDTLS_PARAM_FAILED_ALT */
+void aria_invalid_param( )
+{
+    mbedtls_aria_context ctx;
+    unsigned char key[128 / 8] = { 0 };
+    unsigned char input[MBEDTLS_ARIA_BLOCKSIZE] = { 0 };
+    unsigned char output[MBEDTLS_ARIA_BLOCKSIZE] = { 0 };
+    unsigned char iv[MBEDTLS_ARIA_BLOCKSIZE] = { 0 };
+    size_t iv_off = 0;
+
+    TEST_INVALID_PARAM( mbedtls_aria_init( NULL ) );
+    TEST_VALID_PARAM( mbedtls_aria_free( NULL ) );
```

Comment:
The test of `TEST_VALID_PARAM( mbedtls_aria_free( NULL ) )` has it's own function in AES because it doesn't depend on `MBEDTLS_CHECK_PARAMS` and `!MBEDTLS_PARAM_FAILED_ALT`.

If that logic applies there, it should also apply here.

Reply:
It does. I deliberately kept it in the `MBEDTLS_CHECK_PARAMS`-only test `invalid_params()`, but if you prefer I can add a separate test.

Reply:
I will add a separate test in line with the AES PR.